### DISCUSSION TITLE
Code conventions

### DISCUSSION THREAD

**Discussion by mgm1313** [buchungapp/nationaal-watersportdiploma#40] [Ideas]
### File naming
`kebab-case-for-all-files`
Motivation: OSX is case insensitive, but Linux is. This way, we can prevent any issues with renaming files.

**Comment by mgm1313** [comment]
@elmerbulthuis Right now, we have a naming convention within the `packages/*` and `apps/*` folders using the `nwd-` prefix. For example: `nwd-db` and `nwd-web`. I'd propose we drop this prefix, since it's redundant, we're already within the `nwd` scope. How do you think about this?

**Reply by elmerbulthuis** [reply]
I'm fine with any name! However I am not a fan of renaming things. But I'm very much ok with it.

**Reply by elmerbulthuis** [reply]
noticed that the acronym for nationaal watersportdiplome is nawadi. I will rename things as i spot them (if they are still there)

unless you stop me!

**Reply by mgm1313** [reply]
> noticed that the acronym for nationaal watersportdiplome is nawadi. I will rename things as i spot them (if they are still there)
> 
> 
> 
> unless you stop me!

Yeah 'NWD' was already taken on NPM, so for the scope (@) name I thought of this one 🤪. I think it looks cute, and still easy to remember.
